logical_switch_port.go

```diff
@@ -206,6 +207,28 @@ func (odbi *ovndb) lspSetOptionsImp(lsp string, options map[string]string) (*Ovn
 	return &OvnCommand{operations, odbi, make([][]map[string]interface{}, len(operations))}, nil
 }
 
+func (odbi *ovndb) lspSetDynamicAddressesImp(lsp string, address string) (*OvnCommand, error) {
```

Comment:
Addresses is string [] not just string.
Also add a nil check for at-least lsp as its a mandatory param for this operation. 
Double confirm if addresses is also mandatory and hence no harm in adding nil check for that too.

Reply:
This is for dynamic_addresses column in the `Logical_Switch_Port` table. That is not a `[]string`, but just a `string`.

Reply:
By nil check, you mean checking for empty string? For the lsp name, if someone passes an empty string, I would expect the row fetch to fail. Also, we have a use-case for the dynamic_address field to be set to empty string. So that check isn't needed. Passing an empty string is equivalent of clearing the field.

Reply:
I compared the nbctl commands and it takes just string so we are good.

yes no need to even look in cache for empty string similar to len(lsp)==0 return. We do it for all required entities which lsp in this case.